.gitlab-ci.yml

```diff
@@ -69,3 +73,27 @@ publish:release:
   script:
     - aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.gradle-properties --with-decryption --query "Parameter.Value" --out text > ./gradle.properties
     - ./gradlew :dd-sdk-android:bintrayUpload
+
+notify:release:
+  extends: .slack-notifier-base
+  stage: notify
+  when: on_success
+  only:
+    - tags
+  script:
+    - BINTRAY_URL="https://bintray.com/datadog/datadog-maven/dd-sdk-android"
```

Comment:
why do you need these constant ? It seems not to be used in the script ?

Reply:
Fixed :)